### PR TITLE
feat: Wrap NodeSingleConnected in a View so it can be styled

### PR DESCRIPTION
**Pull request by vinnyjth**
### What does this PR do, or why is it needed

Allows us to fix the "action bar overlapping content" bug in templates. 

Simulator Screen Shot - iPhone 11 - 2021-01-29 at 14 27 22
Simulator Screen Shot - iPhone 11 - 2021-01-29 at 14 27 17

## TODO:

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [tag-issues-here]
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android
- [ ] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

**Comment by redreceipt** [comment]
I'm sorry I don’t understand what's the bug here?

**Comment by vinnyjth** [comment]
This change let’s me add padding to content items that use the ActionBar, so that the content isn’t clipped by the action bar when you scroll to the bottom on non-media content items. On Monday I’ll push up the accompanying change on Templates

**Comment by vinnyjth** [comment]
@redreceipt Here's how it would be used in practice https://github.com/ApollosProject/apollos-templates/pull/251

File: packages/apollos-ui-connected/src/NodeSingleConnected/index.js [hunk 1]
```diff
@@ -30,33 +30,41 @@ const NodeSingleInner = ({ nodeId, ImageWrapperComponent }) => (
     <UpNextButtonConnected nodeId={nodeId} />
     <ContentParentFeedConnected nodeId={nodeId} />
     <ContentChildFeedConnected nodeId={nodeId} />
-  </>
+    <ContentChildFeedConnected nodeId={nodeId} />
+    <ContentChildFeedConnected nodeId={nodeId} />
```
Comment (redreceipt) [inline packages/apollos-ui-connected/src/NodeSingleConnected/index.js:34]: why are there three of these lines?

Comment (vinnyjth) [reply packages/apollos-ui-connected/src/NodeSingleConnected/index.js:34]: 😬

File: packages/apollos-ui-connected/src/NodeSingleConnected/index.js [hunk 2]
```diff
@@ -30,33 +30,41 @@ const NodeSingleInner = ({ nodeId, ImageWrapperComponent }) => (
     <UpNextButtonConnected nodeId={nodeId} />
     <ContentParentFeedConnected nodeId={nodeId} />
     <ContentChildFeedConnected nodeId={nodeId} />
-  </>
+    <ContentChildFeedConnected nodeId={nodeId} />
```
Comment (vinnyjth) [inline packages/apollos-ui-connected/src/NodeSingleConnected/index.js:33]:
```suggestion
```

File: packages/apollos-ui-connected/src/NodeSingleConnected/index.js (hunk 1 above)
Comment (vinnyjth) [inline packages/apollos-ui-connected/src/NodeSingleConnected/index.js:34]:
```suggestion
```

Comment (vinnyjth) [reply packages/apollos-ui-connected/src/NodeSingleConnected/index.js]: I was making the content longer so I could reproduce the issue I was trying to fix.

**Comment by vinnyjth** [comment]
@redreceipt Took care of feedback here (and in templates)

**Review by redreceipt — APPROVED** [review]
